README + tutorial (#11)
paraynaud/PartiallySeparableSolvers.jl: docs/src/index.md

# PartiallySeparableSolvers.jl

## Philosophy

> PartiallySeparableSolvers.jl define partitioned quasi-Newton trust-region solvers for unconstrained `ADNLPModel` or `MathOptNLPModel`.

Reviewer: ```suggestion
PartiallySeparableSolvers.jl implements a partitioned quasi-Newton trust-region solver for unconstrained optimization.
```
